### PR TITLE
feat: ignore the query string when serving client JavaScript

### PR DESCRIPTION
**Pull request by TimWolla**
### The kind of change this PR does introduce

* [ ] a bug fix
* [x] a new feature
* [ ] an update to the documentation
* [ ] a code change that improves performance
* [ ] other

### Current behavior

Client JavaScript will not be served if the request URL contains a query string.

### New behavior

Client JavaScript will be served, even if the request URL contains a query string.

### Other information (e.g. related issues)

Resolves socketio/socket.io#4023

### CODE REVIEW

**Comment by TDCLV** [comment]
Would this be good to end client side interactions?

**Comment by TimWolla** [comment]
@TDCLV Sorry, I don't follow. Can you rephrase / expand on what your question is?

**Review by TDCLV — APPROVED** [review]

**Comment by darrachequesne** [comment]
@TimWolla thanks :+1:

**Comment by TimWolla** [comment]
My pleasure.
